src/language-html/printer-html.js

```diff
@@ -884,7 +883,10 @@ function printOpeningTagStartMarker(node) {
 }
 
 function printOpeningTagEndMarker(node) {
-  assert(!node.isSelfClosing);
+  /* istanbul ignore next */
+  if (node.isSelfClosing) {
+    throw new Error("node.isSelfClosing should be false");
+  }
```

Comment:
:-1: 4 lines instead of 1 and what for? Assertions are an idiomatic way to do the same thing this `if` does. I see no reason to remove them. If they're considered development-only, we should configure the build system to remove them.